apps/atlas/src/app/app-routing.module.ts

```diff
@@ -8,7 +8,7 @@ const routes: Routes = [
     path: 'search',
     loadChildren: () =>
       import('./search/search.module').then((m) => m.SearchModule),
-    data: { title: 'Search', icon: faSearch },
+    data: { title: 'Search', icon: faSearch, hidden: true},

```

Comment:
Perfect, glad you used this option to hide the button rather than removing it outright!

Only nitpick is the formatting of the line. Should look into setting up auto-formatting as part of the `pre-commit` checks.

```suggestion
    data: { title: 'Search', icon: faSearch, hidden: true },
```